src/org/thoughtcrime/securesms/ConversationFragment.java

```diff
@@ -660,6 +693,10 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
           handleSaveAttachment((MediaMmsMessageRecord)getSelectedMessageRecord());
           actionMode.finish();
           return true;
```

Comment:
After this return statement, the function should return. Why you still have case statement?